### DISCUSSION TITLE
Lc0 Backend config in Chessbase UI for dual GPU ( 3090 and 2080ti)

### DISCUSSION THREAD

**Discussion by ALIANS-PRODUCTIONS** [LeelaChessZero/lc0#1672] [Q&A]
What is the correct way to configure a dual gpu backend in the UCI engine window of Chessbase for LC0?

Lc0 version 0.28.2
Chessbase 16 (update 12)
i9 12900k, msi rtx3090 and 2080ti, windows 11, 

I am able to force select either gpu by entering 'gpu=0' or 'gpu=1' in the BackendOptions field.
However the Lc0 blog suggestion of : threads=2,cudnn(gpu=0),cudnn-fp16(gpu=1) doesnt work in the api
And it seems chessbase ingnores any lc0.cfg file. 
Thank you very much in advance for your help!

![lc0 Chessbase](https://user-images.githubusercontent.com/21268921/146093565-d476dfee-60b6-460b-a98e-6abcef830ac0.png)

**Comment by Naphthalin** [comment]
There are several multi-GPU backends, `demux`, `multiplexing` and `roundrobin`. For non-identical GPUs like in your case, only one of them is recommended (multiplexing). Since there are other details in settings, best ask in the discord or in navratil25's twitch chat, since he used a similar setup.

**Comment by courierdotsix** [comment]
backend=multiplexing
backend-opts=(backend=cuda-auto,gpu=0),(backend=cuda-auto,gpu=1)

This worked for me on Chessbase 16.
